test/pummel/test-process-cpuUsage.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+const start = process.cpuUsage();
+
+// process.cpuUsage() should return {user: aNumber, system: aNumber}
+assert(start != null);
+assert(Number.isFinite(start.user));
+assert(Number.isFinite(start.system));
+
+// Run a busy-loop for 2 seconds.
+const now = Date.now();
+while (Date.now() - now < 2000);
```

Comment:
Note this is in the "pummel" tests - I copied from [`test/pummel/test-process-hrtime.js`](https://github.com/nodejs/node/blob/23818c6d0ba5503263f97bcd888b1c227b3177af/test/pummel/test-process-hrtime.js#L12), which also uses 2 sec.

Will go ahead and change to 500ms though, don't see a real advantage of using 2 sec over that.